martini.go

```diff
@@ -57,6 +57,12 @@ func (m *Martini) Action(handler Handler) {
 	m.action = handler
 }
 
+// Action sets the logger
```

Comment:
@mvader 
https://github.com/ovr/martini/blob/70f17d289f9ca20ff7beed9ab501b163b0cd5e42/martini.go#L55
So I named method like 'Logger' via I saw `Action` method (it is another setter)